homeassistant/components/tesla/binary_sensor.py

```diff
@@ -29,6 +29,11 @@ def __init__(self, tesla_device, controller, sensor_type):
         self.entity_id = ENTITY_ID_FORMAT.format(self.tesla_id)
         self._sensor_type = sensor_type
 
+    @property
+    def unique_id(self) -> str:
```

Comment:
We could put this in the `TeslaDevice` class instead and not have to copy it to all the platforms.

Reply:
Good point

Reply:
ha, that's the first thing I did and I didn't see it working :/